addon/debounce-task.ts

```diff
@@ -1,7 +1,8 @@
 import { assert } from '@ember/debug';
 import { cancel, debounce } from '@ember/runloop';
 import { IDestroyable, IMap, EmberRunTimer } from './types';
-import { registerDisposable } from './utils/disposable';
+import { registerDestructor } from 'ember-destroyable-polyfill';
```

Comment:
Should be imported from the official location (not the polyfill module):

```suggestion
import { registerDestructor } from '@ember/destroyable';
```

Reply:
Oh alskdjflaskdf. I think that was vscode doing that shenanigans.